README.md

```diff
@@ -440,6 +440,135 @@ const subscriptionId = '<subscription_id>'; // player id
 await api.endLiveActivity('<app_id>', '<activity_id>', subscriptionId);
 ```
 
+## Users
+### Creating a OneSignal User
+```js
+const user = new OneSignal.User();
+
+const aliasLabel = '<alias_label>';
+const aliasId = '<alias_id>';
+const subscriptionToken = '<subscription_token>';
+
+user.identity = {
+    [aliasLabel]: aliasId,
+};
+
+user.subscriptions = [
+  {
+    type: "iOSPush",
```

Comment:
I think it would be great to make this an importable enum. I think this would require some custom modifications but just throwing it out there

e.g: 
```ts
type: SubscriptionType.IosPush,
```

Reply:
if not included in this release, at least link to a documented list of acceptable types from this section

Reply:
This is an enum, but typescript generator gives it the following type: 
export type SubscriptionObjectTypeEnum = "iOSPush" | "AndroidPush" | "FireOSPush" | "ChromeExtensionPush" | "ChromePush" | "WindowsPush" | "SafariLegacyPush" | "FirefoxPush" | "macOSPush" | "HuaweiPush" | "SafariPush" | "Email" | "SMS" ;
So, if you hover over this type in IDE it will show the hint with all possible values. Nevertheless, I think it's good idea to put it in the readme explicitly.